nautobot/docs/models/dcim/deviceredundancygroups.md

```diff
@@ -0,0 +1,359 @@
+# Device Redundancy Groups
```

Comment:
Sorry to jump in at the last minute but shouldn't this file be named singular, not plural, e.g. `deviceredundancygroup.md`?

Reply:
Yes. Good catch!

Reply:
thx!